client/my-sites/plugins-wpcom/business-plugins-panel.jsx

```diff
@@ -18,17 +19,24 @@ const defaultPlugins = [
 
 export const BusinessPluginsPanel = React.createClass( {
 	render() {
-		const { plugins: givenPlugins = [] } = this.props;
+		const {
+			isActive = false,
+			plugins: givenPlugins = []
+		} = this.props;
 		const plugins = givenPlugins.length
```

Comment:
@artpi we could but @retrofox was touching this line in another PR so I wasn't going to touch it until that's done